src/components/ContentHighlights/ContentHighlightsCardItemsContainer.jsx

```diff
@@ -17,6 +16,7 @@ import { generateAboutPageUrl } from './data/utils';
 import EVENT_NAMES from '../../eventTracking';
 import { features } from '../../config';
 import DeleteArchivedHighlightsDialogs from './DeleteArchivedHighlightsDialogs';
+import { isArchivedContent } from '../EnterpriseApp/data/enterpriseCurationReducer';
```

Comment:
[nit/question]: since we're using this helper method in a couple of places, would it make more sense to pull it out of the `enterpriseCurationReducer` file and put it in a general location?